ansible/docker-files/api.env.j2

```diff
@@ -0,0 +1,42 @@
+# Because defaults are based on local development it is important to explicitly provide values for production.
+
+# The server uses a postgres database to store and retrieve information
+DB_PASSWORD={{ pgpassword }}
+DB_NAME={{ pgdatabase }}
+DB_PORT={{ pgport }}
+DB_HOST={{ pghost }}
+DB_USER_NAME={{ pguser }}
+SQL_IS_USING_SSL={{ sql_is_using_ssl }}
+
+# Test users can be enabled to allow login in local dev environments. This will bypass auth0
+AUTH_USE_TEST_USERS={{ auth_use_test_users }}
+
+# The environment {dev|staging|prod}
+ENV={{ env }}
+
+# This can be set to production (not sure if this is actually used)
+MODE=
+
+# The public location of the frontend service (for example https://infinite.industries)
+APP_URL={{ app_url }}
+
+DOMAIN=
+
+# The Port to run the application on
+PORT={{ api_port }}
+
+# Slack webhooks are used to send messages about events on the website to a slack channel (they are not required)
+SLACK_WEBHOOK_EVENT_SUBMISSION={{ slack_webhook_event_submission }}
+SLACK_WEBHOOK_VENUE_SUBMISSION={{ slack_webhook_venue_submission }}
+
+# Bitly is used to provided shortened urls for sharing events to social media
+#BITLY_TOKEN="401cbf1121b4859d3a0c8f09b623193eb0233fac"
```

Comment:
This probably shouldn't be hardcoded here, in fact this is considered a secret I'm pretty sure. We may want to cycle this, though that said I feel like we may have stopped using bitly